gui/wxpython/gui_core/pyedit.py

```diff
@@ -400,6 +412,28 @@ def OnOpen(self, event):
         """Handle open event but ask about replacing content first"""
         if self.CanReplaceContent('file'):
             self.Open()
+            self.recent_files.AddFileToHistory(
+                filename=self.filename,
+            )
+
+    def OpenRecentFile(self, path, file_exists):
+        """Try open recent file and read content
+
+        :param str path: file path
+        :param bool file_exists: file path exists
+
+        :return: None
+        """
+        if self.CanReplaceContent(by_message='file'):
+            if not file_exists:
+                GError(
+                    _("File path <{}> doesn't exist.".format(path)),
```

Comment:
I would say "File <{}> does not exist. It was probably moved or deleted."

I think the classic style would be not to include period if it is just once sencence. However, I don't know if this still holds. There is general tendency to use multiple sentences and proper style which would be an argument for a period. I ended up suggesting two seances anyway because I think we should at least give suggestions on what might have happen when we cannot give suggestion on what to do next (how to fix it).

Another consideration would be existing messages, but surprisingly, there is not many of these sentence in the code. An older "file not found" is more prevalent (but considered wrong and confusing now, sounds like "the software was searching for that file").

Reply:
All right, I fixed it.